repository-data/application/src/main/resources/hcm-content/content/documents/administration/valuelists/documenttypes.yaml

```diff
@@ -31,3 +31,15 @@
       jcr:primaryType: selection:listitem
       selection:key: hee:blogPost
       selection:label: BlogPost
+    /selection:listitem[6]:
```

Comment:
Could we reload this value-list via `hcm-actions.yaml` ?

Reply:
Added